src/main/java/seedu/address/model/TeachingAssistantBuddy.java

```diff
@@ -248,7 +248,7 @@ public void removeStudent(Student key) {
 
     @Override
     public String toString() {
-        return students.asUnmodifiableObservableList().size() + " persons";
+        return students.asUnmodifiableObservableList().size() + " students";
         // TODO: refine later
```

Comment:
What do we need to refine here?

Reply:
not sure tbh I'll just remove the comment ahhaha thanks!!